### DISCUSSION TITLE
[Effective Java] - 읽기 전 참고자료

### DISCUSSION THREAD

**Discussion by Irisation23** [cs-collections/cs-collections-discussion#5] [Java]
### 번역 용어 해설 📝
- [이펙티브 자바 3판 번역 용어 해설](https://docs.google.com/document/d/1Nw-_FJKre9x7Uy6DZ0NuAFyYUCjBPCpINxqrP0JFuXk/edit)

### 깃허브 저장소 💿
- [한국어판 깃허브 저장소](http://git.io/fAm6s) - _해당 코드의 주석에는 책 페이지가 언급되어 있습니다._

**Comment by JoisFe** [comment]
테스트 댓글 입니다.

**Comment by ahngilwoong** [comment]
네 알겠습니다 (뇌)
